pyccel/stdlib/ndarrays/ndarrays.c

```diff
@@ -7,18 +7,75 @@
 # include <string.h>
 # include <stdarg.h>
 # include <stdlib.h>
+# include <stdio.h>
+
+/*
+ * Takes an array, and prints its elements the way they are laid out in memory (similar to ravel)
+*/
+
+void print_ndarray_memory(t_ndarray nd)
+{
+    int i = 0;
+    while (i < nd.length)

```

Comment:
Any reason to prefer while rather than for?

Reply:
No, I'll change it to for

Reply:
https://github.com/pyccel/pyccel/pull/1128/commits/c061f83fb9860c823980c3bb90039d78d368c480